Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -203,6 +203,90 @@ namespace
         return Napi::Persistent(jsInputSource);
     }
 
+    xr::DetectionBoundary CreateDetectionBoundary(const Napi::Object& object)
```

Comment:
Would it be better for this to take in an xr::DetectionBoundary&, and just fill the values on the object that way rather than creating a new DetectionBoundary object then copying the values over after returning?

Reply:
this seems like a good suggestion

---
In reply to: [540498049](https://github.com/BabylonJS/BabylonNative/pull/476#discussion_r540498049) [](ancestors = 540498049)